packages/ra-ui-materialui/src/layout/UserMenu.js

```diff
@@ -36,23 +39,35 @@ class UserMenu extends React.Component {
     };
 
     render() {
-        const { children, label, logout, translate } = this.props;
+        const {
+            children,
+            label,
+            customIcon,
+            icon,
+            logout,
+            translate,
+        } = this.props;
         if (!logout && !children) return null;
         const { anchorEl } = this.state;
         const open = Boolean(anchorEl);
 
+        const menuIconProps = {
+            'arial-label': label && translate(label, { _: label }),
```

Comment:
We should provide a default label too imo. Something like `Profile` ?

Reply:
Currently there is a default label used as default prop: `ra.auth.user_menu`.

Reply:
Oh sorry missed it